azure_functions_worker/loader.py

```diff
@@ -11,13 +11,14 @@
 import pathlib
 import sys
 import typing
+from os import PathLike, fspath
```

Comment:
just curious: is this ordering change needed or just a tidying up change?

Reply:
Just tidying up changes - more like PyCharm and PEP8 corrected it.